src/theme/typography.ts

```diff
@@ -12,7 +12,7 @@ import {
 import { BASE_FONT_SIZE, BASE_LINE_HEIGHT } from './constants';
 
 const family: Family = {
-  base: "'proxima-nova', 'Lato', 'Helvetica Neue', Arial, Helvetica, sans-serif",
+  base: "'Inter', 'Lato', 'Helvetica Neue', Arial, Helvetica, sans-serif",
```

Comment:
This is not enough :) more typography-related changes in https://github.com/securityscorecard/ssc-ui-core/blob/e006babf3860b98e04e58e5345f7b77da45086bc/src/styles/ambrosia/index.js#L30 are missing from this PR